docs/development/i18n.rst

```diff
@@ -1,437 +1,461 @@
+``i18n_tool.py`` file must be manually edited to add this new language
+to the ``SUPPORTED_LANGUAGES`` variable.
 
-Updating Strings to be Translated
----------------------------------
+Update strings to be translated
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-After strings are modified in the code, templates, JavaScript or desktop
-labels, the ``securedrop/translations/messages.pot`` files must also be
-updated. Individual developers should NOT do this when changing strings
-in the code; the translations will be updated in bulk later on.
+After strings are modified in the SecureDrop source code, templates or
+desktop labels, the ``securedrop/translations/messages.pot`` files
+must also be updated. Individual developers should not do this
+whenever changing strings in the code; the translations are updated in
+bulk when it's time to update the Weblate fork.
 
 Translations can be updated with the following command:
 
 .. code:: sh
 
-    make translate
+    $ make translate
 
 This wraps ``i18n_tool.py translate-messages`` and ``i18n_tool.py
-translate-desktop``.  These commands will update
-``securedrop/translations/messages.pot``,
-``install_files/ansible-base/roles/tails-config/templates/desktop.pot``,
-and the ``messages.po`` files for each language.
+translate-desktop``.  These commands update the `.pot`_ files for the
+SecureDrop server code and the desktop icons, as well as the `.po`_
+files for each language.
 
-.. note:: The changes will only be visible in the `Weblate`_ web
-   interface used by translators after :ref:`merging_develop_into_the_weblate_fork`.
+.. note:: The new source strings will only be visible to translators
+    in `Weblate`_ after the ``develop`` branch is merged into the
+    Weblate fork.
 
-Compiling Translations
-----------------------
+.. _merge_develop_to_weblate:
 
-`gettext`_ needs a compiled file for each language (the ``*.mo``
-files). This can be done by running the following command:
+Merge develop into the Weblate fork
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code:: sh
+`Weblate`_ works on a long standing fork of the `SecureDrop git
+repository`_ and is exclusively responsible for the content of the
+``*.pot`` and ``*.po`` files. The content of the ``develop`` branch
+must be merged into the ``i18n`` branch to make updated source strings
+available to Weblate.
 
-    securedrop/bin/dev-shell ./i18n_tool.py --verbose translate-messages --compile
+Translation must be suspended in `Weblate`_, and any uncommitted
+changes committed and pushed, to avoid conflicts:
 
-For desktop files the compilation phases creates a modified version of
-the original file which includes all the translations collected from
-the ``.po`` files.
+* Go to the `Weblate repository page for SecureDrop`_.
 
-This can be done by running the following command::
+|Weblate commit Lock|
 
-    securedrop/bin/dev-shell ./i18n_tool.py --verbose translate-desktop --compile
+* Click ``Commit``.
+* Click ``Push``.
```

Comment:
@rmol I would love to keep the text from last PR on this (as in develop).

```
* Click ``Push`` (we do these two steps, so that uncommitted changes
  in the repo do not create merge conflicts.)
```

Reply:
I moved the explanation before the two steps, just above. Do you want this in addition to that, or do you mean you'd prefer that it just be explained here?